client/src/components/Details.jsx

```diff
@@ -24,14 +24,14 @@ class Details extends React.Component {
     let readMore = null;
     let icon = faChevronDown;
     if(this.state.showReadMore === true) {
-      readMore = <ReadMore />
+      readMore = <ReadMore gettingAround={this.props.gettingAround} />
       icon = faChevronUp;
     }
     return (
       <div>
         <div onClick={this.handleReadMoreClick}>
           <span className="readMore">Read more about the neighborhood</span>
-          <FontAwesomeIcon icon={icon} />
+            <FontAwesomeIcon className="chevron" icon={icon} />
```

Comment:
<FontAwesomeIcon (line 34) is indented one more space over than it needs to be. Bring it back one.